### ISSUE TITLE
Support bundling with `esbuild`

### ISSUE THREAD

**Issue by ryan-token** [Storyboard-fm/little-media-box#18]
Bundling with [esbuild](https://esbuild.github.io/) currently fails on this package and other libs that use this package like `hypercast-podcore-next` due to an issue in the [batch](https://github.com/visionmedia/batch) dependency.

On [line 15 of Batch's `index.js`](https://github.com/visionmedia/batch/blob/47a9b1d1f5cd8e4b32dbb74a52aa968cf9b58480/index.js#L15) it references a package called "emitter" that does not exist. When esbuild tries to resolve that package, it fails.

One solution: fork the Batch repo, work around that failed resolver, and publish that simple change as its own package.

**Comment by ryan-token** [comment]
@agrathwohl My updates were merged into the main Batch repo with [this commit](https://github.com/visionmedia/batch/commit/8feb5ff9bcb683598308970f54b43a53e9417e6e). So Batch can now be built with esbuild!

Now just waiting on the maintainer to cut a new release to NPM that includes these changes. Once that's done, all we'll need to do here is install the latest version of Batch and cut a new release of little-media-box that the serverless repo can update to.
